components/bookshelf/Navigation.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react"
+import { NavBox, NavList, Link } from "../styled/bookshelf"
+
+const Navigation = ({ active, setActive }) => {
```

Comment:
can you add a `NavigationProps`?

```
interface NavigationProps {
 active: boolean;
 setActive: React.Dispatch<React.SetStateAction<boolean>>;
}

const Navigation = ({ active, setActive }: NavigationProps) => {
```
Either that or you can use the global state, `Context API`

Reply:
![image](https://user-images.githubusercontent.com/91857581/188588229-7c2f89d5-c0ba-44a3-8862-0c43d2ba01e2.png)